scout/parse/variant/frequency.py

```diff
@@ -59,6 +60,15 @@ def parse_frequencies(variant, transcripts):
             frequencies["gnomad_max"] = gnomad_max
             break
 
+    # For mitochondrial variants, keep both "hom" and "het" freqs
+    gnomad_mt_hom = parse_frequency(variant, "GNOMAD_MT_AF_HOM")
+    if gnomad_mt_hom is not None:
```

Comment:
```suggestion
    if gnomad_mt_hom:
```

Reply:
I was just sticking to the theme. Every other place in this file uses `is not None`. I could change it for all of them though, but I don't feel like rocking the boat too much :)

Reply:
Ok, but then let's stick to is not None, as it's also less prone to give unwanted results!